starter-code/css/main.css

```diff
@@ -0,0 +1,86 @@
+body{
+	font-family: Lato;
+	font-weight: normal;
+	color: black;
+	padding-left: 50px;
+	border-left: rgba(0,0,0,.4) 5px solid;
+	margin: 0 25% 0 25%;
+	padding-top: 2%;
+	padding-bottom: 2%;
+}
+article img{
+	width: 100%
+}
+h1 a, h2 a{
+	text-decoration: none;
+	font-weight: normal;
+	color:black;
+}
+h1 a{
+	font-size: 50px;
+	color: orange;
+}
+h2{
+	font-size:35px;
+	font-weight: normal;
+}
+header nav{
+	font-size: 25px;
+}
+header nav a{
+	color: rgba(0,0,0,.2);
+	text-decoration: none;
```

Comment:
Since you are writing out `text-decoration: none;` more than once, consider adding a CSS property to all anchors from the start, and then elimination that repetition in your CSS.